src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata minClientVersion="2.8.1">
     <id>Microsoft.DotNet.BuildTools</id>
-    <version>1.0.25-prerelease</version>
+    <version>1.0.26-prerelease</version>
```

Comment:
The build process will automatically append the "dash daily build number" to the package version so no need to include it directly.